tests_requre/database/test_models.py

```diff
@@ -599,6 +599,18 @@ def test_get_project(clean_before_and_after, a_copr_build_for_pr):
     assert project.project_url == "https://github.com/the-namespace/the-repo-name"
 
 
+def test_get_forge(clean_before_and_after, multiple_forge_projects):
+    projects = GitProjectModel.get_forge(0, 10, "github.com")
```

Comment:
oki, here's my answer :D 

so will `https://github.com` work as well?

Reply:
no, this will not work, as in other similar methods